### DISCUSSION TITLE
WiGUI is malware?

### DISCUSSION THREAD

**Discussion by Raff-117** [Romanitho/Winget-AutoUpdate#473] [General]
WiGUI is being reported by Kaspersky as a high-risk trojan malware!

**Comment by Romanitho** [comment]
Very true, it is a powershell script converted to an '.exe' via the well known ps2exe tool. Everything is totally open source:
https://github.com/Romanitho/Winget-Install-GUI
The wigui.exe builder:
https://github.com/Romanitho/Winget-Install-GUI/blob/main/.github/workflows/Create_New_Version.yaml

This thing is, some AV detect ps2exe builds as malware. False positive, but detected.
If that scares you, simply run the powershell script. You'll have the same GUI.

**Reply by Raff-117** [reply]
I thought it might be a false positive and reported it to see if it's something in the code that caused the false alert to be generated. Since all the code is available, this gives you peace of mind. Sorry for the over-the-top.

**Reply by Romanitho** [reply]
no, it's ok. I wanted to have all theses tools full open, for several reasons, but mainly for security :)
Exe is just there to simplify things for everyone.
